docs/_articles/en/codebuilder/about.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Code Builder Overview
+lang: en
+---
+## Overview
+Salesforce Code Builder is a web-based integrated development environment that has all the power and flexibility of Visual Studio Code, Salesforce Extensions for VS Code, and Salesforce CLI in your web browser. Code Builder provides a modern developer experience for all developers, regardless of expertise level. Code Builder makes it easy for admins and developers to seamlessly collaborate in the cloud without having to worry about downloading software, setup or your machine specs.
+
+The Code Builder dev environment that can be spun up in seconds and is custom to your Salesforce org and your specified project. Through Code Builder, access your favorite Salesforce languages and frameworks, such as Apex, SOQL, Visualforce, Aura, and Lightning Web Components. Access rich developer tools such as debuggers for Apex, Linting etc.
```

Comment:
I'm not sure what this first sentence means. I get it that you can spin up a CB dev environment in seconds, but I don't quite understand the second part.

Reply:
(Style rule) Also, use "and so on" instead of Latin phases such as "etc, i.e"